src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -972,7 +969,7 @@ private List<OffenderSentenceDetailDto> offenderSentenceSummaries(final String a
     }
 
     private List<OffenderSentenceDetailDto> offenderSentenceSummaries(final String agencyId, final Set<String> caseloads, final boolean filterByCaseloads) {
-        final var query = buildAgencyQuery(agencyId, authenticationFacade.getCurrentPrincipal());
+        final var query = buildAgencyQuery(agencyId, hmppsAuthenticationHolder.getUsername());
```

Comment:
switch to `username` as must be a nomis user to get case loads (which the build eventually calls)